saleor/graphql/checkout/mutations/checkout_lines_update.py

```diff
@@ -55,6 +56,15 @@ class CheckoutLineUpdateInput(BaseInputObjectType):
         required=False,
     )
 
+    metadata = NonNullList(
+        MetadataInput,
+        description=(
+            f"Checkout line public metadata. Will add and update keys. To delete keys use deleteMetadata mutation. {ADDED_IN_321} "
```

Comment:
> Will add and update keys

To keep it consistent, shouldn't we update all metadata input descriptions? Right now this can be misleading. For example, here we explicitly say that this is `add and update`, but we don't say that in `orderUpdate` mutation. This can suggest, that `checkoutLinesUpdate` works in a different way than the rest of the metadata inputs

Reply:
> To keep it consistent, shouldn't we update all metadata input descriptions?

(or just create a task, I am not suggesting to extend the scope)

Reply:
I will create a task, I think we need to scan a code - and I believe it's still better to leave better description in one, than to be consistent but not saying enough

Reply:
https://linear.app/saleor/issue/EXT-2072